tiatoolbox/utils/__init__.py

```diff
@@ -1,13 +1,4 @@
 """Utils package for toolbox utilities."""
-from tiatoolbox.utils import (
```

Comment:
why these have been removed?

Reply:
This was removed by ruff

Reply:
it shouldn't remove unused imports from __init__

Reply:
If you want to import modules just to expose them to other modules I believe __all__ should be used. Hence why ruff removed them, because they appear unused.

Reply:
I haven't touched any imports. Any changes are from ruff / pre-commit

Reply:
They now show up as unused but are no longer being removed.